### ISSUE TITLE
can't open shapes from visium dataset

### ISSUE THREAD

**Issue by kevinyamauchi** [scverse/napari-spatialdata#65]
I was just testing the [napari region annotation notebook](https://spatialdata.scverse.org/en/latest/tutorials/notebooks/notebooks/examples/napari_rois.html) and I can't open the associated shapes data. I can however open the image data

![Screenshot 2023-05-04 at 16 58 20](https://user-images.githubusercontent.com/1120672/236247120-8c2d59e7-b46e-4832-ab6e-9869dc1f134f.png)

### reproduce
1. start the [napari region annotation notebook](https://spatialdata.scverse.org/en/latest/tutorials/notebooks/notebooks/examples/napari_rois.html) 
2. load the data as specified in the notebook and launch `Interactive()`
3. I see the coordinate systems and the elements
4. If I double click the image element, the image is loaded (see screenshot above)
5. Double clicking the shapes doesn't do anything

### installation
```
conda create -n test-sd python=3.9
conda activate test-sd
pip install spatialdata[extra] notebook
```

napari-spatialdata 0.2.1
spatialdata

**Comment by kevinyamauchi** [comment]
@giovp , do you see the same issue?

**Comment by giovp** [comment]
@rahulbshrestha could you take a look? we discussed at previous metting

**Comment by kevinyamauchi** [comment]
closed y #70
